src/org/usfirst/frc4904/robot/commands/RunOuttakeMotors.java

```diff
@@ -0,0 +1,19 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class RunOuttakeMotors extends CommandGroup {
+	
+	public RunOuttakeMotors() {
+		requires(RobotMap.Component.ballIO);
+		addParallel(new MotorConstant(RobotMap.Component.ballIO.topMotor, -1.0));
```

Comment:
Want to move these constants to the class (static) instead of in the constructor body?